dev-libs/libcec: Fix udev rule [please reassign]

alanswanson opened the pull request:
Also change udev rule to set group ownership to "video" instead of "uucp" as matches common usage and group membership, typically Kodi or other media player.

Closes: https://bugs.gentoo.org/765358
Closes: https://bugs.gentoo.org/788742
Signed-off-by: Alan Swanson <reiver@improbability.net>

ceamac commented:
Hi!

This package became maintainer-needed a long time ago, but was still assigned to the previous maintainer.

If you are still interested in this, please update it to EAPI 8 and python 3.11 and append `[please reassign]` to the PR title so it will get assigned to @proxy-maint.

Thank you!

ceamac commented:
`[please reassign]` should be appended at the end of the title; I think the Gentoo bot ignores it at the beginning.

ceamac commented:
Thanks, I'll review it this evening

ceamac reviewed (changes requested):
Looks good, with only a few minor issues.

With `USE="cubox exynos kernel-cec python tools udev"`

```
 * VDB: detected possibly incorrect RDEPEND (dev-libs/libcec-6.0.2-r1)
 * sys-libs/ncurses:= < 
 *                    > x11-libs/libX11
```

Please check if `sys-libs/ncurses` really is needed (it might be), and please add `x11-libs/libX11` to `RDEPEND` (and check if it is needed with certain USE flags or always)

Thank you!

alanswanson commented:
Updated ebuild per suggested changes.

> ```
>  * VDB: detected possibly incorrect RDEPEND (dev-libs/libcec-6.0.2-r1)
>  * sys-libs/ncurses:= < 
>  *                    > x11-libs/libX11
> ```

That's inherited from the current ebuild which has implicit dependency for x11-libs/libX11 via x11-libs/libXrandr. Added x11-libs/libX11 to RDEPEND.

> Please check if `sys-libs/ncurses` really is needed (it might be), and please add `x11-libs/libX11` to `RDEPEND` (and check if it is needed with certain USE flags or always)

Actually, no. It supports "curses" not "ncurses" and fails on checking for libcurses per bug https://bugs.gentoo.org/51901. Just removed the dependency.

alanswanson commented on dev-libs/libcec/libcec-6.0.2-r1.ebuild:
```diff
@@ -0,0 +1,125 @@
+src_install() {
+	cmake_src_install
+
+	use python && python_optimize "${D}$(python_get_sitedir)"
```
Had to revert this suggestion as caused QA failure; _DoublePrefixInPath ${ED}$(python_get_sitedir): concatenates two paths containing EPREFIX on line: 113_

ceamac reviewed (approved):
Great!

Thank you!
